sphinx_gherkindoc/cli.py

```diff
@@ -160,6 +161,20 @@ def process_args(
         verbose(f"Writing sphinx glossary: {glossary_filename}")
         glossary.write_to_file(glossary_filename)
 
+    if tag_list_name:
+        taglist_dox = make_tag_list(doc_project)
+        if args.dry_run:
+            verbose("No taglist generated")
+            return
+
+        if taglist_dox is None:
+            print("No tags to include in the tag list: no tag list generated")
+            return
+
+        taglist_filename = output_path / f"{tag_list_name}.rst"
+        verbose(f"Writing sphinx tag list: {taglist_filename}")
+        taglist_dox.write_to_file(taglist_filename)
+
```

Comment:
This code block looks like it was copied from the glossary block right above it.
Being that the previous block has 'return' statements in it, the code might never get to this block.
And if another block is added after this one, a return here might prevent the code from getting to that.
This means restructuring the two blocks. My initial feeling is that this function is already too big :-) but that is a pre-existing condition for this change.  I think in the interests of simplicity, I'm willing to leave this as is unless you want to undertake refactoring the glossary and this new tags into helper functions.